tests/unit/RegisterStateDeductions.cpp

```diff
@@ -59,9 +59,7 @@ BOOST_AUTO_TEST_CASE(GPRArgumentsAndVRReturnValue) {
   abi::Definition::RegisterSet RValues{ VRRetValues[0] };
   RValues = ABI.enforceReturnValueRegisterState(std::move(RValues));
   revng_check(RValues.contains(VRRetValues[0]));
-  revng_check(not RValues.contains(VRRetValues[1]));
   revng_check(not RValues.contains(GPRRetValues[0]));
-  revng_check(not RValues.contains(GPRRetValues[1]));
```

Comment:
I think I've a fix for these too, but OK.

Reply:
I guess we both were running tests under asan in parallel, no wonder we ran into the same issues.